ArjunaJTS/idl/idlj/src/main/ant/gen-stubs-jdk9.xml

```diff
@@ -1,206 +0,0 @@
-<project name="Narayana IDL stubs for jdk9">
```

Comment:
@jmfinelli I think this should go in a commit attributed to JBTM-3590 changes since JDK 9 is no longer supported with that - is that OK with you?

For some history, it looks like it was added in https://github.com/jbosstm/narayana/commit/4e4a477e49feeb3784f3b469387964ff4dd8d29c but it looks like it's potential use was commented out (at least in the two files I found it was mentioned in ArjunaJTS/idl/idlj/idlj-jdk8/pom.xml and ArjunaJTS/idl/idlj/idlj-jdk9/pom.xml).

Reply:
I can add a commit to remove `ArjunaJTS/idl/idlj/src/main/ant/gen-stubs-jdk9.xml` (mentioning JBTM-3590). WDYT?

Reply:
+1 - thanks

Reply:
Thanks Tom :-)